gpytorch/variational/cholesky_variational_distribution.py

```diff
@@ -61,3 +44,8 @@ def variational_distribution(self):
         # Now construct the actual matrix
         variational_covar = CholLazyTensor(chol_variational_covar)
         return MultivariateNormal(self.variational_mean, variational_covar)
+
+    def initialize_variational_distribution(self, prior_dist):
+        self.variational_mean.data.copy_(prior_dist.mean)
```

Comment:
Didn’t this once exist on Strategy instead of Distribution, because whitened vs unwhitened initialization was different? I take it we assume whitened now?

Reply:
`VariationalStrategy.prior_distribution` returns `N(0, I)`, so this works in both the whitened and unwhitened cases. Also, this way we don't have to re-define `kl_divergence` for VariationalStrategy which is nice.